src/src/rx_main.cpp

```diff
@@ -346,7 +347,7 @@ bool ICACHE_RAM_ATTR HandleSendTelemetryResponse()
     Radio.TXdataBuffer[0] |= (crc >> 6) & 0b11111100;
     Radio.TXdataBuffer[7] = crc & 0xFF;
 
-#if defined(LBT_ACTIVE)
+#if defined(Regulatory_Domain_EU_CE_2400)
     if(ChannelIsClear())
```

Comment:
This whole if/else block should be wrapped like
```
if (LBTEnabled)
{
    ...
}
else
{
    Radio.TXnb();
}
```

Reply:
Same as above?